changelog/changelog.py

```diff
@@ -78,19 +78,38 @@ def callback(response, *args, **kwargs):
             hooks={'response': callback}
         )
 
-    def get_latest_milestone(self):
+    def fetch_open_milestones(self):
         """
-        Get info about the latest milestone currently open in issues.
+        Fetch list of milestone dicts. Each dict contains metadata about an open milestone.
         """
         url = f"{self.api_url_prefix}/repos/{self.repo_url}/milestones"
-        r = self.request(url).json()
-
-        logger.debug(f"Open milestones found: {r}")
-        if len(r) == 0:
+        open_milestones = self.request(url).json()
+        if not open_milestones:
             raise ValueError("No open milestone was found on github.")
-        elif len(r) > 1:
-            logger.info(f"Found multiple open milestones on github. Using latest: {r[0]}")
-        return r[0]
+        logger.debug(f"Open milestones found: {open_milestones}")
+        return open_milestones
+
+    def get_most_recently_updated_milestone(self):
+        """
+        Get info about the most recently updated milestone.
+        """
+        open_milestones = self.fetch_open_milestones()
+        milestone = sorted(open_milestones, key=lambda m: m['updated_at'])[-1]
+        logger.info(f"Using most recently updated milestone: '{milestone['title']}'")
+        return milestone
```

Comment:
Previously, this was `get_latest_milestone`, but "latest" was ambiguous (see #22), so I've tried to make it's actual functionality clearer.